content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -88,6 +88,147 @@ Once the installation is finished, RDI is ready for use.
 
 {{<note>}}RDI gives you instructions to help you create secrets and create your pipeline.{{</note>}}
 
+## "Silent" installation
+
+You can use the
+[`redis-di install`]({{< relref "/integrate/redis-data-integration/ingest/reference/cli/redis-di-install" >}})
+command with the `--file` option (or the shorter version `-f`) to supply answers
+to the installer's questions automatically from a [TOML](https://toml.io/en/) file:
+
+```bash
+redis-di install --file silent.toml
+```
+
+The following example shows the properties you can use:
+
+```toml
+title = "RDI Silent Installer Config"
+
+high_availability = false
+scaffold = false
+deploy = true
```

Comment:
deploy is when you want RDI to deploy the config.yaml immediately after installation. this might be useful for R&D but not for our users. so we should remove from example and reference
IMO scaffold should be true in the example

Reply:
Done.